### ISSUE TITLE
install script asks for CUPS 2.4 but it is installed

### ISSUE THREAD

**Issue by Nachtpfoetchen** [michaelrsweet/lprint#114]
Hello,
I try to get the source going to investigate further for [this issue](https://github.com/michaelrsweet/lprint/issues/113).

What i did:
git clone and cd
then i tried to execute ./configure with user permission.

```
~/lprint (master)> ./configure
[shorted for simplicity]
configure: error: CUPS 2.4 or later is required for LPrint.
```

When i look for the CUPS version that i have installed:
```
aptitude show cups
Package: cups                            
Version: 2.4.2-3+deb12u5
State: installed
```

Is this a bug, or is something with my setup wrong?

My system: Debian bookwrom (stable)

**Comment by michaelrsweet** [comment]
Do you have the libcups2-dev package installed?

**Comment by Nachtpfoetchen** [comment]
yes, i do.

```
aptitude show libcups2-dev
Package: libcups2-dev                    
Version: 2.4.2-3+deb12u5
State: installed
```

**Comment by michaelrsweet** [comment]
OK, can you provide a copy of the complete "configure" output and the config.status file?

**Comment by Nachtpfoetchen** [comment]
here is the complete output:
```
~/lprint (master)> ./configure
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether gcc accepts -g... yes
checking for gcc option to enable C11 features... none needed
checking for ranlib... ranlib
checking for ar... /usr/bin/ar
checking for mkdir... /usr/bin/mkdir
checking for rm... /usr/bin/rm
checking for install-sh script... using /home/alice/lprint/install-sh
checking for pkg-config... /usr/bin/pkg-config
configure: error: CUPS 2.4 or later is required for LPrint.
```
The config.status was cp to config.status.md, because otherwise github prevents the file upload.
[config.status.md](https://github.com/michaelrsweet/lprint/files/13932645/config.status.md)

**Comment by michaelrsweet** [comment]
Sorry, I meant the config.log file. Alternatively what does `pkg-config --libs cups` report?

**Comment by Nachtpfoetchen** [comment]
pkg-config returns an error [1].
The log is in this comment.
```
pkg-config --libs cups
Package cups was not found in the pkg-config search path.
Perhaps you should add the directory containing `cups.pc'
to the PKG_CONFIG_PATH environment variable
Package 'cups', required by 'virtual:world', not found
```
[config.log](https://github.com/michaelrsweet/lprint/files/13932676/config.log)

**Comment by michaelrsweet** [comment]
Oh crap, Ubuntu/Debian's libcups2 packages are missing the pkg-config file... 😡

**Comment by michaelrsweet** [comment]
Wasn't fixed until the 2.4.6 packages in mantic...

**Comment by michaelrsweet** [comment]
[master f22b455] Fix configure on older Ubuntu distros (Issue #114)
